product_tier_validation/tests/test_product_tier_validation.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 Open Source Integrators
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common, tagged
+
+
+@tagged("-at_install", "post_install")
+class TestProductTierValidation(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        group_ids = cls.env.ref("base.group_system").ids
+        cls.test_user_1 = cls.env["res.users"].create(
```

Comment:
Is it possible to re-use the users created in the base_tier_validation module?

Can you disable the email password reset and tracking. ( Like in base module )